zds/utils/mps.py

```diff
@@ -103,7 +103,7 @@ def send_message_mp(
         signals.new_content.send(sender=post.__class__, instance=post, by_email=send_by_mail,
                                  no_notification_for=no_notification_for)
 
-    if send_by_mail and direct:
+    if send_by_mail or direct:
```

Comment:
J'ai du modifier ce bout de code car un test échouait : 

```
FAIL: test_sanctions (zds.member.tests.tests_views.MemberTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/zestedesavoir/zds-site/zds/member/tests/tests_views.py", line 763, in test_sanctions
    self.assertEqual(len(mail.outbox), 1)
AssertionError: 0 != 1
```

Depuis la PR #5649 qui améliore le fonctionnement des sanctions données aux membres, les courriels notifiant des sanctions ne sont plus envoyés (c'est pourquoi ce test échoue). On cherche à la fois à notifier le membre (`direct=False`) et envoyer le courriel (`send_by_mail=True`) ce qui n'est pas possible avec la condition `send_by_mail and direct` que j'ai transformée en `send_by_mail or direct` qui est selon moi ce que l'on attend de cette fonction.